do.py

```diff
@@ -4,6 +4,93 @@
 import sys
 import shutil
 import subprocess
+import platform
+
```

Comment:
python do.py test -> should also run go unit test and fail if coverage is lower than certain number.

check https://github.com/open-traffic-generator/openapiart/pull/105

Reply:
addressed